packages/react-core/src/components/Form/__tests__/__snapshots__/FormGroup.test.tsx.snap

```diff
@@ -1,679 +1,35 @@
-<div>
-  <div
-    class="pf-c-form__group"
-  >
-    <div
-      class="pf-c-form__group-label"
-    >
-      <label
-        class="pf-c-form__label"
-        for="label-id"
-      >
-        <span
-          class="pf-c-form__label-text"
-        >
-          Label
-        </span>
-      </label>
-       
-    </div>
-    <div
-      class="pf-c-form__group-control"
-    >
-      <input
-        id="label-id"
-      />
-      <div>
-        label
-      </div>
-    </div>
-  </div>
-</div>
-`;
+exports[`FormGroup should render form group variant with function helperText 1`] = `"<div data-testid=\\"form-group-test-id\\" class=\\"pf-c-form__group\\"><div class=\\"pf-c-form__group-label\\"><label class=\\"pf-c-form__label\\" for=\\"label-id\\"><span class=\\"pf-c-form__label-text\\">Label</span></label> </div><div class=\\"pf-c-form__group-control\\"><input id=\\"label-id\\"><div>label</div></div></div>"`;
 
-exports[`FormGroup component should render form group variant with function label 1`] = `
-<div>
-  <div
-    class="pf-c-form__group"
-  >
-    <div
-      class="pf-c-form__group-label"
-    >
-      <label
-        class="pf-c-form__label"
-        for="id"
-      >
-        <span
-          class="pf-c-form__label-text"
-        >
-          <div>
-            label
-          </div>
-        </span>
-      </label>
-       
-    </div>
-    <div
-      class="pf-c-form__group-control"
-    >
-      <input
-        aria-label="input"
-      />
-      
-    </div>
-  </div>
-</div>
-`;
+exports[`FormGroup should render form group variant with function label 1`] = `"<div data-testid=\\"form-group-test-id\\" class=\\"pf-c-form__group\\"><div class=\\"pf-c-form__group-label\\"><label class=\\"pf-c-form__label\\" for=\\"id\\"><span class=\\"pf-c-form__label-text\\"><div>label</div></span></label> </div><div class=\\"pf-c-form__group-control\\"><input aria-label=\\"input\\"></div></div>"`;
 
-exports[`FormGroup component should render form group variant with node helperText 1`] = `
-<div>
-  <div
-    class="pf-c-form__group"
-  >
-    <div
-      class="pf-c-form__group-label"
-    >
-      <label
-        class="pf-c-form__label"
-        for="label-id"
-      >
-        <span
-          class="pf-c-form__label-text"
-        >
-          Label
-        </span>
-      </label>
-       
-    </div>
-    <div
-      class="pf-c-form__group-control"
-    >
-      <input
-        id="label-id"
-      />
-      <h1>
-        Header
-      </h1>
-    </div>
-  </div>
-</div>
-`;
+exports[`FormGroup should render form group variant with node helperText 1`] = `"<div data-testid=\\"form-group-test-id\\" class=\\"pf-c-form__group\\"><div class=\\"pf-c-form__group-label\\"><label class=\\"pf-c-form__label\\" for=\\"label-id\\"><span class=\\"pf-c-form__label-text\\">Label</span></label> </div><div class=\\"pf-c-form__group-control\\"><input id=\\"label-id\\"><h1>Header</h1></div></div>"`;
```

Comment:
again no `.pf-c-form__helper-text`

Reply:
This time I do see the helperText, where in this test the prop is a div with the text "Header" which does exist here.